src/main/java/com/checkmarx/jenkins/CxScanBuilder.java

```diff
@@ -1242,10 +1287,19 @@ private void mapSastConfiguration(Optional<SastConfig> sast, CxScanConfig scanCo
     }
 
 
-    private void createScaReports(AstScaResults scaResults, File checkmarxBuildDir) {
+    private void createScaReports(AstScaResults scaResults, File checkmarxBuildDir, @Nonnull FilePath workspace) {
```

Comment:
i dont see use of new argument workspace in this method. Kindly check if it is required.